Caboodle/DataTransfer/DataTransfer.android.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Android.Content;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class DataTransfer
+    {
+        public static Task ShowShareUI(ShareTextRequest request)
```

Comment:
Should this be `UI` or `Ui`?

Reply:
I followed what UWP did... I think UI is correct.

Reply:
Because it is 2 words is my thought here.